tests/testthat/_snaps/as_polars_lf.md

```diff
@@ -0,0 +1,21 @@
+# as_polars_lf.default throws an error
+
+    Code
+      as_polars_lf(1)
+    Condition
+      Error in `as_polars_df()`:
```

Comment:
It is confusing that the call is `as_polars_lf()` but the message points to `as_polars_df()`

Reply:
I see what you mean, but since `as_polars_lf.default()` is a thin wrapper around `as_polars_df()`, as the documentation states, I'm wondering if it's worth doing another `tryCatch` for the error message.

Reply:
I think it's worth it since it only needs to be implemented in `as_polars_lf.default`. It would be different if we needed an extra tryCatch() in several methods.